src/dependencies.js

```diff
@@ -21,13 +21,20 @@ const dependencyMap = {
   xtst: '(libXtst or libXtst6)'
 }
 
+async function spawnRpmBuild (logger) {
+  return await spawn('rpmbuild', ['--version'], logger)
+}
 /**
  * Retrieves the RPM version number and determines whether it has support for boolean
  * dependencies (>= 4.13.0).
  */
 async function rpmSupportsBooleanDependencies (logger) {
-  const output = await spawn('rpmbuild', ['--version'], logger)
-  return rpmVersionSupportsBooleanDependencies(output.trim().split(' ')[2])
+  const output = await module.exports.spawnRpmBuild(logger)
+
+  const parts = output.trim().split(' ')
+  const versionNumber = parts.slice(-1)[0]
```

Comment:
```suggestion
  const versionNumber = _.last(parts)
```

And add `const _ = require('lodash')` at the top, in alphabetical order.